src/sentry/api/endpoints/organization_releases.py

```diff
@@ -30,7 +30,13 @@
 from sentry.utils.cache import cache
 
 
-ERR_INVALID_STATS_PERIOD = "Invalid stats_period. Valid choices are '', '24h', and '14d'"
+SUMMARY_STATS_PERIOD = ("24h", "48h", "7d", "14d")
```

Comment:
Could this be pulled from the other definition in snuba/sessions?

Reply:
I will want to refactor this a bit going forward. There is some duplication all around.